interview-questions.md

```diff
@@ -8,50 +8,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
-Researched answer:
+Researched answer: Functions and methods are both used to perform a set of instructions. The biggest difference between the two is that methods are assosciated with an object, where functions are not. There are also a variety of actions that are so commonly performed by developers, JavaScript developers have added them into the language itself, createing what is called a 'built-in method'. .split() is a built in method that turns a string into an array, includes() is a built in method that acts on an array and returns a boolean statement as to whether or not that array contains a specific value. This makes methods incredibly effecient and smooth to work with. However, there are times when you need to do something that is not assosciated directly with a specific object, but instead needs to act on multiple objects, or even just perform functions such as manipulation of data or outputting data. In this case, you will need to create a function, which will allow you to code in anything that is needed for your specific task. While functions have vast flexibility, truly efficient code will make good use of the many built in methods as much as possible. 
 
 6. STRETCH: What is hoisting in JavaScript?
 
-Your answer:
+Your answer: (Based only on my knowledge of the word, this ought to be fun) JavaScript is a fairly well described language, so I would assume that hoisting changes the 'elevation' of an object. Elevation in this sense would refer moving an object from where it is to a different part of the code, for example, if an object such as a function is located deep in your code, you can 'hoist' it upward to use it earlier in your code. This would allow using a function earlier in the code that it is written, but would likely be considered bad practice, as properly ordering your code is more important than relying on hoisting. 
 
-Researched answer:
+Researched answer:Typically in JavaScript,  Hoisting is a process in which the 'interpreter' (a piece of the browser that runs through the code and interprets it) acts as if the declaration of certain objects to the top of their scope. Effectively, there are certain pieces of code that are 'lifted' to the top of it's scope. This can be understood as the interpreter effectively reading your code as if you had placed hoisted objects at the first line of it's applicable scope. An example of this is the var decleration, it has a global scope and is hoisted, which means even if you have a line of code at the very bottom of your code such as var orange = 'fruit' you could start your code with console.log(orange) would return 'fruit' even though it hadn't been declared yet. 
 
```

Comment:
Great job on all these interview questions and furthering your understanding of these concepts by really diving into the researched answers!